config-example.yml

```diff
@@ -255,6 +255,10 @@ introspection:
   # Default: true
   hideSubscriptionsWithUndocumentedReturnType: true
 
+  # Hide any Types that are not used anywhere in your schema after other things have been hidden.
+  # Default: true
+  hideUnusedTypes: true
```

Comment:
This value was defaulted to `false` in the microfiber instance created in the `augmenters.js`, but since no options were passed to the microfiber instance in `src/themes/default/data/index.js`, then unused types were being removed automatically already.

So putting it to `true` here should not be a breaking change, except for those who maybe implemented their own theme and did something else in the `src/themes/default/data/index.js` microfiber instance.